js/friday.js

```diff
@@ -1,34 +1,34 @@
-function onLoad() {
-    let today = new Date().getDay();
-    let element = document.getElementById("friday");
-    if (today === 5) {
-      element.innerHTML = "YES";
-      let month = new Date().getMonth();
+(function IsItFridayYet() {
+  const date = new Date();
+  const theMonth = date.getMonth();
+  const theDay = date.getDay();
+  const element = document.getElementById("friday");
+  const spans = document.getElementsByClassName("fallingElement");
+  let season = "";
 
-      let season = "";
-      switch (true) {
-        case month >= 3 && month < 6:
-          season = "spring";
-          break;
-        case month >= 6 && month < 9:
-          season = "summer";
-          break;
-        case month >= 9 && month < 12:
-          season = "autumn";
-          document.body.style.backgroundColor = "fff7c8";
-          break;
-        default:
-          season = "winter";
-          document.body.style.backgroundColor = "dcf0f4";
-          break;
-      }
-      let spans = document.getElementsByClassName("fallingElement");
-      Array.from(spans).forEach(function (element) {
-        element.classList.add(season);
-      });
-    } else {
-      element.innerHTML = "NO";
+  if (theDay === 5) {
+    element.innerHTML = "YES";
+
+    switch (true) {
+      case theMonth >= 3 && theMonth < 6:
+        season = "spring";
+        break;
+      case theMonth >= 6 && theMonth < 9:
+        season = "summer";
+        break;
+      case theMonth >= 9 && theMonth < 12:
+        season = "autumn";
+        document.body.style.backgroundColor = "fff7c8";
+        break;
+      default:
+        season = "winter";
+        document.body.style.backgroundColor = "dcf0f4";
+        break;
     }
+    Array.from(spans).forEach(function (element) {
+      element.classList.add(season);
+    });
+  } else {
+    element.innerHTML = "NO";
   }
-
-onLoad();
+})();
```

Comment:
Now the big one. I have removed the invocation of `onLoad();` for the pattern of an immediately invoked function expression, or an IFFE. As it sounds, by using an IFFE, we don't need to call onLoad. It can just run the minute the file loads or its triggered by something. The biggest advantage of an IFFE is the following:

Assuming you didn't already merge this PR, on your browser, I want you to visit your [Friday site](https://isitfridayyet.today/). Open the console, and type `onLoad`

Did you see? Your onLoad function is exposed! Gasp! This exposure isn't that big a deal because you can just download any of the source files on this project but the point here is that we want to obscure any leaks whenever possible.

however, you wrapped this entire onLoad function into an IFFE `()();` It is inaccessible through the console -FYI: 
```
(your whole parent function code goes here) (this remains empty);
```

Along with that, because it is a parent function that will be in charge of all the code on the site, we can give it a capital lettered name like `OnLoad` to drive home what a big deal this function is. We can go a step further and call this the `Jess` function or the `IsItFridayFridayYet` function.

**Extra credit:** You can make an IFFE an annonymous function, but I do not recommend it (nor do I recommend anonymous functions in general outside of a few cases). Why? Well, peep this. if we decide that our IFFE is going to be anonymous, we would write it like this: `(() => {})();` instead of `(function blah{})();`.

If hypothetically, we introduced a bug in the code on an anonymous function, the console would produce an error that read like this...

```
// in VSCode
(()=> {
const date = new Date();
const theMonth = date.getMonth();
const theDay = date.getDay();

...

shitsFuckedUp;

})();

// Now if we see the error on the browser, we would get back this on the console.

Uncaught error ...
at thisFile.js
```

if the parent function is named:

```
// in VSCode
(IsItFridayYet() {
const date = new Date();
const theMonth = date.getMonth();
const theDay = date.getDay();

...

shitsFuckedUp;

})();

// Now if we see the error on the browser, we would get back this on the console.

Uncaught error ...
at IsItFridayYet (thisFile.js)
```